src/frontend/js/components/SearchInput/index.spec.tsx

```diff
@@ -11,13 +11,13 @@ describe('<SearchInput />', () => {
   const inputProps = {};
 
   it('renders with the input field and button', () => {
-    const { container, getByText } = render(
+    const { container, getByTestId } = render(
       <IntlProvider locale="en">
         <SearchInput context={contextProps} inputProps={inputProps} />
       </IntlProvider>,
     );
 
-    getByText('Search');
+    getByTestId('search-input-btn');
```

Comment:
As the button is now accessible, it could be relevant to try to find it though its accessible role and name ? WDYT?

`getByRole('button', { name: 'Search' })`

Reply:
It's indeed much cleaner that my approach.